pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoRunSample.java

```diff
@@ -0,0 +1,17 @@
+package ca.on.oicr.pinery.lims.miso;
+
+import ca.on.oicr.pinery.lims.DefaultRunSample;
+
+public class MisoRunSample extends DefaultRunSample {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)